src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -170,6 +170,7 @@ void OnSystemRequestNotification::AddHeader(BinaryMessage& message) const {
   }
 
   char timeout_str[24];
+  // cppcheck-suppress invalidPrintfArgType_sint
   if (0 > sprintf(timeout_str, "%d", timeout)) {
```

Comment:
Please do not supress. Fix code issue

Reply:
Fixed in 9a5fd54723f107d621b976d7fb98f746b208812b
Please check